oauth2/org.wso2.dpop.proof.generator/src/main/java/org/wso2/dpop/proof/generator/DPOPProofGenerator.java

```diff
@@ -0,0 +1,235 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```

Comment:
```suggestion
 *  Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
```

Reply:
fixed with https://github.com/wso2/samples-is/pull/486/commits/340ccf50cbf8a8c486e9b43d2438f001348050a1